pkg/operator/builder.go

```diff
@@ -37,9 +38,12 @@ import (
 )
 
 const (
-	DefaultAtlasDomain      = "https://cloud.mongodb.com/"
-	DefaultSyncPeriod       = 3 * time.Hour
-	DefaultLeaderElectionID = "06d035fb.mongodb.com"
+	DefaultAtlasDomain           = "https://cloud.mongodb.com/"
+	DefaultSyncPeriod            = 3 * time.Hour
+	DefaultIndependentSyncPeriod = 15 * time.Minute
+	DefaultLeaderElectionID      = "06d035fb.mongodb.com"
+
+	minimumIndependentSyncPeriod = 5 * time.Minute
```

Comment:
I think this should be a param here and a constant in main. That would still restrict production but not our tests.
WDYT?